phoenicis-javafx/src/test/java/org/phoenicis/javafx/AppConfigurationInitialisationTest.java

```diff
@@ -29,10 +40,17 @@ public static void setupHeadless() {
     }
 
     @Test
-    public void testAppConfigurationInitialisation() throws TimeoutException {
-        setupHeadless();
-        FxToolkit.registerPrimaryStage();
-        FxToolkit.setupFixture(() -> new AnnotationConfigApplicationContext(AppConfiguration.class));
+    public void testAppConfigurationInitialisation() {
+        try {
+            setupApplicationRoot();
+            setupHeadless();
+            FxToolkit.registerPrimaryStage();
+            FxToolkit.setupFixture(() -> new AnnotationConfigApplicationContext(AppConfiguration.class));
+        } catch (TimeoutException | IOException e) {
```

Comment:
Why don't we continue to forward the exceptions to junit?

Reply:
Codacy was complaining that the test didn't contain `fail` or `assert`.